src/jit/codegenarmarch.cpp

```diff
@@ -555,6 +555,16 @@ void CodeGen::genPutArgStk(GenTreePutArgStk* treeNode)
         {
             genConsumeReg(source);
             emit->emitIns_S_R(storeIns, storeAttr, source->gtRegNum, varNumOut, argOffsetOut);
+            if (compiler->opts.compUseSoftFP && targetType == TYP_LONG)
```

Comment:
`compUseSoftFP` was introduced when we first ported armel. The value is TRUE when `ARM_SOFTFP` is defined, FALSE otherwise(for all architectures). Since then I use `compUseSoftFP` if applicable. However some modules like LSRA which do not hold `compiler` object I just use `#ifdef ARM_SOFTFP`.